app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split(" ")
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names